tests/handlers/records-read.spec.ts

```diff
@@ -1614,6 +1917,49 @@ export function testRecordsReadHandler(): void {
       });
     });
 
+    describe('createFilter()', async () => {
+      it('should not throw if only `recordId` is set', async () => {
```

Comment:
Sanity: This method never throws. Why test that it doesn't throw under these conditions?